docs/doxygen/overviews/xrc_format.h

```diff
@@ -1221,6 +1231,36 @@ page.
 @endTable
 
 
+@subsubsection xrc_wxinfobar wxInfoBar
+
```

Comment:
I'd either mention that all the sub-elements are optional or explicitly add "Optional" and/or "default" to each element description, to make it clear that they can all be omitted.